safe_control_gym/experiments/base_experiment.py

```diff
@@ -405,7 +413,7 @@ def compute_metrics(self, data, verbose=False):
         }
         return metrics
 
-    def get_episode_data(self, key, postprocess_func=lambda x: x):
+    def get_episode_data(self, key, postprocess_func=lambda x: x, exponentiate=False):
```

Comment:
Should add this `exponentiate` argument to the Args list. But more importantly, what is this and why is it needed?

Reply:
If it is only for returns, why not just create a different postprocess_func?